docs/asciinema/openqa-in-5-minutes.yaml

```diff
@@ -0,0 +1,102 @@
+---
```

Comment:
Where can I find a documentation of this file format? I searched https://docs.asciinema.org/ but could not find anything

Reply:
And this is where I was able to find it initially: https://docs.asciinema.org/integrations/?h=autocast